go.mod

```diff
@@ -2,6 +2,12 @@ module github.com/FoundationDB/fdb-kubernetes-operator
 
 go 1.19
 
+replace github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.12.1
```

Comment:
Have you checked which dependencies require the old versions? Maybe it's also valid to just update those dependencies instead adding the replace statements. Have you used any open-source tool to get the notifications of those CVEs?

Reply:
This was based on scanning with trivy (https://github.com/aquasecurity/trivy), but we made a mistake and based the scan results on v1.8.1, that is why these CVEs refer to older go versions. We will update the PR based on v1.11.0 scan results.

Reply:
Thanks for doing that! Just wanted to know which tool was used to see if we can integrate that in our pipelines :)